etsin_finder/frontend/js/components/qvain/utils/getReferenceData.js

```diff
@@ -3,3 +3,19 @@ import axios from 'axios';
 const getReferenceData = (referenceData) => axios.get(`https://metax.fairdata.fi/es/reference_data/${referenceData}/_search?size=1000`)
 
 export default getReferenceData;
+
+export const getLocalizedOptions = (field) => getReferenceData(field).then(res => {
+  const hits = res.data.hits.hits
```

Comment:
This should be fixed to use the `uri` field of the JSON as the `value` in the selects.